RunningOrder/Sprint/SprintsView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SprintsView.swift
+//  RunningOrder
+//
+//  Created by Clément Nonn on 07/07/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+
+extension Sprint: Identifiable {
+    var id: String { self.name }
```

Comment:
would it be possible to have two sprints with the name? What would happen in such case?

Reply:
To me it is not possible to have the same sprint name. Maybe I should check at creation of the sprint :)